internal/netif/netif.go

```diff
@@ -66,6 +87,13 @@ func (m *netifManagerDefault) EnsureIPAddress() error {
 
 	klog.Infof("Successfully added %q to %q", m.addr.String(), m.devName)
 
+	if m.manageInterface {
+		err = m.LinkSetUp(l)
+		if err != nil {
+			return fmt.Errorf("could set link up: %v", err)
+		}
+	}
+
```

Comment:
Why do it here instead of when the device is created? Is this even necessary?

Reply:
I'm not sure but when adding a new link, it should not be `up`. So we explicitly set it as `up`